backend/battles/views.py

```diff
@@ -14,6 +15,15 @@ def get_form_kwargs(self):
         kwargs["current_user"] = User.objects.first()
         return kwargs
 
+    def get_success_url(self):
+        return reverse("battles:battle_update_opponent_pokemons", kwargs={"pk": self.object.pk})
+
+
+class BattleUpdateOpponentPokemons(UpdateView):
```

Comment:
Doesn't this need a `get_success_url`?

Reply:
Well, the default behavior for `UpdateView` seems to be the redirection for the `DetailView`. And this is correct for my current context. So, everything looks fine for now. What do you think about it?

Reply:
Aha! 
```
You don’t even need to provide a success_url for CreateView or UpdateView - they will use get_absolute_url() on the model object if available.
```
https://docs.djangoproject.com/en/3.1/topics/class-based-views/generic-editing/#model-forms

Cool!

Reply:
Yes, that's it! I was looking for the documention too, but I didn't managed to find it.